src/main/content/developer-tools.html

```diff
@@ -5,124 +5,243 @@
+                        <a href="https://www.docker.com" target="_blank" aria-label="Link to docker site">Docker (Docker Compose for Mac)</a>
+                    </span>
                 </div>
-            </li>
-            <li>
-                <p>Follow the wizard and accept licenses.</p>
-            </li>
-            <li>
-                <p>After installation is complete, restart Eclipse.</p>
-            </li>
-            <li>
-                <p>Navigate to Codewind view.</p>
-                <code>Window > Show View > Other > Codewind > Codewind Explorer</code>
-            </li>
-            <li>
-                <p>Install Docker Images.</p>
-                <p>Double-click on the Codewind item in the explorer to install. This may take a few minutes.</p>
-            </li>
-            <li>
-                <p>Create a new project or work with an existing one.</p>
-                <p>Codewind creates a folder called codewind-workspace within your home directory (<code>C:\codewind-workspace</code> on Windows) to contain your projects</p>
-            </li>
-        </ol>
-        <p class="col-md-12">Need more help using Codewind? <a href="https://www.eclipse.org/codewind/docindex.html" target="_blank" class="cpa-link" aria-label="Link to Codewind docs">Learn More</a></p>
-    </div>
-    <div class="row content-row hidden-row" id="cli">
-        <p>The Kabanero uses Appsody as its CLI experience to access collections</p>
-        <ol class="col-md-12">
-            <li>
-                <p>Install XCode Command Line Tools and Homebrew.</p>
-                <div class="code-block">
-                    <code>xcode-select --install</code>
+            </div>
+
+            <!-- Detail VS Code Steps -->
+            <div class="row">
+                <div class="col-md">
+                    <ol>
+                        <li>
+                            <p>
+                                Install Codewind in VS Code.
+                                Note: You must have VS Code installed first. If you do not, please visit the
+                                <a target="_blank" href="https://code.visualstudio.com/download">VS Code Download</a> page and download VS Code.
+                            </p>
+                            <a href="vscode:extension/IBM.codewind" role="button" class="btn kabanero-colored-button" 
+                            aria-label="Link to Codewind for Visual Studio Code install">Install Codewind</a>
+                        </li>
+                        <li>
+                            <p>Open the Codewind workspace and create a new project.</p>
+                            <img class="img-fluid image-max" src="/img/dev-tools-new-codewind.png">
+                        </li>
+                        <li>
+                            <p>Select the project template.</p>
+                            <img class="img-fluid image-max" src="/img/dev-tools-template.png">
```

Comment:
The page has horizontal scroll on mobile with the `VS Code` tab selected because of the `image-max` class.  When I removed that class, the horizontal scroll went away but it also gets bigger on desktop view.